docs/make.jl

```diff
@@ -4,7 +4,7 @@
 #  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import Pkg
-Pkg.pkg"add Documenter#2fabe9c3bd40b6288c3d86d96c6031728b2d92ad"
+Pkg.pkg"add Documenter#afeb75d1b2abc7166752c5e00675c8b62ee0bea3"
```

Comment:
@odow, could we try again with https://github.com/JuliaDocs/Documenter.jl/pull/2247/commits/b8ea2c45f09b2205c71b56518714b4ea7e0b44b6 ?

```suggestion
Pkg.pkg"add Documenter#b8ea2c45f09b2205c71b56518714b4ea7e0b44b6"
```